src/main/java/com/mraof/minestuck/data/tag/MinestuckBlockTagsProvider.java

```diff
@@ -22,6 +22,7 @@
 import java.util.concurrent.CompletableFuture;
 
 import static com.mraof.minestuck.block.MSBlocks.*;
+import static com.mraof.minestuck.block.MSBlocks.CINDERED_STAIRS;
```

Comment:
why a static import of this?

Reply:
intellij does that sometimes, will delete